Source/Orts.Simulation/Simulation/RollingStocks/MSTSWagon.cs

```diff
@@ -1660,105 +1665,185 @@ private void UpdateTrainBaseResistance()
 
                 if (IsLowSpeed)
                 {
-                    // If weather is freezing, then starting friction will be greater until bearings have warmed up.
-                    // Chwck whether weather is snowing
-
-                    int FrictionWeather = (int)Simulator.WeatherType;
-                    bool IsSnowing = false;
-
-                    if (FrictionWeather == 1)
-                    {
-                        IsSnowing = true;  // Weather snowing - freezing conditions
-                    }
 
                     // Dtermine the starting friction factor based upon the type of bearing
 
-                    float StartFrictionLow = 0.0f;
-                    float StartFrictionHigh = 0.0f;
+                    float StartFrictionLowLoad = 0.0f;
+                    float StartFrictionHighLoad = 0.0f;
 
```

Comment:
Please describe the two above variables.

Reply:
Ok, done.